easystacks/software.eessi.io/2023.06/a64fx/eessi-2023.06-eb-4.9.4-2023a.yml

```diff
@@ -150,15 +158,103 @@ easyconfigs:
         from-commit: 0437ff1ad34283398f55d4a6e01e6540b1ae9688
 # PR 19646 was included since EB 4.9.1
 #  - snakemake-8.4.2-foss-2023a.eb:
-#      # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19646
 #      options:
+#        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19646
 #        from-pr: 19646
   - snakemake-8.4.2-foss-2023a.eb
 # PRs 19471 and 3036 were included since EB 4.9.1
 #  - LAMMPS-2Aug2023_update2-foss-2023a-kokkos.eb:
-#      # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19471
-#      # see https://github.com/easybuilders/easybuild-easyblocks/pull/3036
 #      options:
+#        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19471
 #        from-pr: 19471
+#        # see https://github.com/easybuilders/easybuild-easyblocks/pull/3036
 #        include-easyblocks-from-pr: 3036
   - LAMMPS-2Aug2023_update2-foss-2023a-kokkos.eb
+## PR 19573 was included since EB 4.9.1
+##  - PyTorch-2.1.2-foss-2023a.eb:
+##      options:
+##        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19573
+##        from-pr: 19573
+#  - PyTorch-2.1.2-foss-2023a.eb
```

Comment:
@trz42 Just pointing out that this one is still commented out, perhaps goal is to tackle this one with a separate PR?